installation_and_upgrade/ibex_install_utils/install_tasks.py

```diff
@@ -89,15 +89,14 @@ def remove_all_and_install_client_and_server(self):
             "config/schema without any extra steps. Proceed?")
 
         self._system_tasks.user_confirm_upgrade_type_on_machine('Client/Server Machine')
+        swap_back_to_old_galil: bool = self._system_tasks.select_galil_driver()
```

Comment:
I might be misunderstanding how `remove_old_ibex` works but will if we run `select_galil_driver` before `remove_old_ibex` any default GALIL IOC set will be removed anyways?

Reply:
This may be a question for @FreddieAkeroyd. I am a bit confused about what is exactly needed when, I thought I had it right before but your comments sounds correct to me

Reply:
Before ibex is removed/reinstalled,  a decision needs to be made as to whether the old or new galil driver will be used.  So this logic sequence is correct is correct (call selct galil driver, install, call swap_gfalil_to_old) but what needs adjusting is some of the logic in select_galil_driver itself (see below)

Reply:
logic updated - select_galil_driver() just need to determine the choice, but not do any actual file manipulation